### PR TITLE
Added Git Question

### PR DESCRIPTION
**Pull request by Aman-Singh-Kushwaha**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

I have added a question on the default text editor used in git bash

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -534,6 +534,16 @@ const gitQuiz = [
       "git commit --amend  is the command used to change the latest commit message.",
     Link: "https://forum.freecodecamp.org/t/git-guide-how-to-amend-your-most-recent-git-commit-message/13186"
   },
+  {
+    Question: "Which one of the following is Git Bash default text editor?",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:538]:
```suggestion
    Question: "Which one of the following is the default text editor in Git Bash?",
```

File: src/data/git-quiz.ts [hunk 2]
```diff
@@ -534,6 +534,16 @@ const gitQuiz = [
...
+    Answer: "Vim",
+    Distractor1: "Emacs",
+    Distractor2: "Nano",
+    Distractor3: "VS Code",
+    Explanation:
+      "Vim is improved version of Vi, was one of popular terminal based text editor among linux users. It can be opened by command: vi fileName",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:544]:
```suggestion
      "Vim stands for Vi Improved and it is terminal based text editor used among Linux users. You can open the editor by using the following command: vi fileName",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Aman-Singh-Kushwaha** [comment]
@jdwilkin4 Sorry for that but, I am quite a beginner and have gone through this mistake 

I was merging updates for which I mistakenly discarded commits due to merge conflict and have to rewrite the same question. And when I looked back on PR, I could not make PR in the same. So I made a new PR #608

**Comment by jdwilkin4** [comment]
No worries, I went ahead and approved your other PR 👍

**Pull request closed without merging**